bff/samples/Hosts.Tests/TestInfra/AppHostFixture.cs

```diff
@@ -35,7 +31,7 @@ public IDisposable ConnectLogger(WriteTestOutput output)
 
     public bool UsingAlreadyRunningInstance { get; private set; }
     public string StartupLogs => _startupLogs.ToString() ?? string.Empty;
-    protected DistributedApplication? App = null!;
+    protected DistributedApplication? App;
     private Logger _logger = null!;
```

Comment:
Take the opportunity to arrange these? e.g. fields, ctor, properties, methods etc.